src/RVRP_data.jl

```diff
@@ -56,35 +58,51 @@ mutable struct SpecificProduct # an entity to understand as a commodity in a mul
     index::Int # Not given in JSON
-    shipment_type::Int # 1 = pickup, 2 = delivery, 3 = pickup&delivery, 4 = complex_request
     product_id::String  
     is_optional::Bool  # default is false
     price_reward::Float64 # if is_optional
-    product_quantity::Float64 # of the shipment/pickup/delivery
+    product_quantity::Float64 # of the shipment
     load_capacity_conso::Float64 # portion of the vehicle load capacity used by the request (it can be equal to the quantity, or different even in terms of unit: volume versus weight for instance).
     split_fulfillment::Bool  # true if split delivery/pickup is allowed, default is false
-    precedence_restriction::Int # default is 0 = no restriction; 1 after all pure pickups, 2 after all pure deliveries, 3 if cannot follow a product that is in the prohibited predecessor list
-    alternative_pickup_point_ids::Vector{String} # defined only if it is a subset of SpecificProduct pickup sources; it is a singleton for request_type = pickup or pickup&delivery
-    alternative_delivery_point_ids::Vector{String} # defined only if it is a subset of SpecificProduct delivery destinations; it is a singleton for request_type = delivery or pickup&delivery
-    setup_service_time::Float64 # optional; on top of PickupPoint service_time; used to measure pre-cleaning or loading time for instance
-    setdown_service_time::Float64 # optional; on top of DeliveryPoint service_time; used to measure post-cleaning or unloading time for instance
+    precedence_restriction::Int # default is 0 = no restriction; 1 after all pure pickups, 2 after all pure deliveries, 3 if cannot follow a product that is in the prohibited predecessor list; 
+    alternative_pickup_point_ids::Vector{String} # defined only if it is a subset of SpecificProduct pickup options; empty otherwise
+    alternative_delivery_point_ids::Vector{String} # defined only if it is a subset of SpecificProduct delivery options; empty otherwise
+    pickup_service_time::Float64 # optional; on top of PickupPoint service_time; used to measure pre-cleaning or loading time for instance
+    delivery_service_time::Float64 # optional; on top of DeliveryPoint service_time; used to measure post-cleaning or unloading time for instance
     max_duration::Float64 # used for the dail-a-ride model or similar applications
 end
 
+struct UnitPrices # cost per unit
+    travel_distance_price::Float64
+    travel_time_price::Float64
+    service_time_price::Float64
+    waiting_time_price::Float64
+end
+
 mutable struct VehicleCategory
     id::String
     index::Int # Not given in JSON
     fixed_cost::Float64
     unit_pricing::UnitPricing
     compartment_capacities::Vector{Float64} # the santard case is to have a single compartment
-    loading_option::Int # 0 = no restrictions, 1 = one request per compartment, 2 = removable compartment separation (note that product conflicts are measured within a compartment)
+    fuel_capacity::Float64
+    reloading_time::Float64
```

Comment:
I think we should have `min_reloading_time` and `max_reloading_time`.  Then, `RecharginPoint.reloading_rate` would mean `fuel_capacity` brought to the vehicle per unit of time.